### PR TITLE
LPS-122029 TikaException: Failed to close temporary resources

### PR DESCRIPTION
**Pull request by IstvanD**
Dear Team,

Could you please review this pull request?

I had to patch the third-party library [isoparser](https://github.com/tballison/mp4parser).
isoparser is used by Tika to parse the contents of MP4 files.

Thank you and best regards,
István

### CODE REVIEW

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by IstvanD** [comment]
Dear Team Lima,

I was wondering if you could find the time to review the pull request.
Please let me know if you need any help or explanation.

Thank you and best regards,
István

File: modules/third-party/isoparser/src/main/java/org/mp4parser/BasicContainer.java [hunk 1]
```diff
@@ -113,6 +113,8 @@ public void initContainer(ReadableByteChannel readableByteChannel, long containe
                 } else {
                     throw e;
                 }
+            } catch (RuntimeException e) {
```
Comment (adolfopa) [inline modules/third-party/isoparser/src/main/java/org/mp4parser/BasicContainer.java:116]: I don't like ignoring such a huge range of errors. Isn't there any more specific error that we can catch? Even better, is there any place in portal where we can catch the error and handle it properly?

Thanks!

Comment (IstvanD) [reply modules/third-party/isoparser/src/main/java/org/mp4parser/BasicContainer.java:116]: Hi @adolfopa if the error is caught in the portal, the temporary file will not be closed, and the issue will persist. I can maybe add logging in the catch block or extract the cause from the RuntimeException and only returning when there is a specific case. Please let me know if you like these ideas. Thanks!

Comment (adolfopa) [reply modules/third-party/isoparser/src/main/java/org/mp4parser/BasicContainer.java:116]: Yes, if there's no other choice, if we could at least log the error I'll be fine with the fix.

Comment (IstvanD) [reply modules/third-party/isoparser/src/main/java/org/mp4parser/BasicContainer.java:116]: OK, I added a logging statement.
https://github.com/liferay-lima/liferay-portal/pull/985

**Pull request closed without merging**